ui/raidboss/data/05-shb/raid/e5s.js

```diff
@@ -7,6 +7,206 @@
   },
   timelineFile: 'e5s.txt',
   triggers: [
+    {
+      id: 'E5S Stratospear Summons',
+      regex: Regexes.ability({ id: '4BA5', source: 'Ramuh', capture: false }),
+      regexDe: Regexes.ability({ id: '4BA5', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.ability({ id: '4BA5', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.ability({ id: '4BA5', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.ability({ id: '4BA5', source: '라무', capture: false }),
+      delaySeconds: 5,
+      preRun: function(data) {
+        data.firstSpear = data.firstSpear || 1;
```

Comment:
If you only need the first stratosphere summons, I think it would be less code and more readable to use a boolean here, rather than also needing a conditional in infoText, e.g.
```
condition: function(data) { return data.seenFirstSpear; },
run: function(data) { data.seenFirstSpear = true; }
```